modules/oc-by-example-content.adoc

```diff
@@ -1856,11 +1856,40 @@ Log in to a server
   # Log in to the given server through a browser
   oc login localhost:8443 --web --callback-port 8280
 
+ifdef::openshift-dedicated,openshift-rosa[]
   # Log in to the external OIDC issuer through Auth Code + PKCE by starting a local server listening port 8080
-  oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8080
+  oc login --exec-plugin=oc-oidc --issuer-url=<issuer_url> --client-id=<client_id> --extra-scopes=email,profile --callback-port=8080
+
+  # Log in with an external OIDC if the external OIDC certificate is not publically trusted
+  oc login --exec-plugin=oc-oidc --issuer-url=<issuer_url> --client-id=<client_id> --extra-scopes=email --callback-port=8080 --oidc-certificate-authority <CA for external OIDC certificate>
+endif::openshift-dedicated,openshift-rosa[]  
 ----
 
+ifdef::openshift-dedicated,openshift-rosa[]
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|`--exec-plugin`
+|Specifies the type of exec plugin credentials used to authenticate the external OIDC issuer. Currently, only `oc-oidc` is supported.
+
+|`--issuer-url`
+|Issuer URL for the external issuer. Required.
+
+|`--client-id`
+|Client ID for the external OIDC issuer. Only supports Auth Code and PKCE. Required.
+
+|`--extra-scopes`
+|Extra scopes for the external OIDC issuer. Optional.
+
+|`--callback-port`
+|The port that the callback server is redirected to after authentication flow is complete. The default is any random, open port.
 
+|`--oidc-certificate-authority`
+|Path to a certificate file for the certificate authority.
```

Comment:
Thanks for many updates so far. I can nearly add LGTM but seems one comment https://github.com/openshift/openshift-docs/pull/74184#discussion_r1576060139 was not fully checked :) i.e. the suggested adding of "external OIDC" was missed :) .
```suggestion
|Path to a certificate file for the external OIDC certificate authority.
```
After this, PR can go forward to be merged, could you add one more update?

Reply:
While @jneczypor adds this, @xingxingxia, I noticed that the `--exec-plugin` argument only supports `oc-oidc` currently. Does that mean that `oc login --exec-plugin=oc-oidc --issuer-url=<issuer_url> --client-id=<client_id> --extra-scopes=email --callback-port=8080 --oidc-certificate-authority <CA for external OIDC certificate>` command does not work on external OIDCs yet, like I am attempting to do here: https://github.com/openshift/openshift-docs/pull/73839?

Reply:
@EricPonvelle The "`oc login --exec-plugin=oc-oidc --issuer-url=<issuer_url> --client-id=<client_id> --extra-scopes=email --callback-port=8080 --oidc-certificate-authority <CA for external OIDC certificate>`" command is exactly for use for external OIDC clusters. `oc-oidc` is exactly for external OIDCs.